src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-react/tests/jsx-ban-props.test.ts

```diff
@@ -0,0 +1,62 @@
+import { convertJsxBanProps } from "../jsx-ban-props";
+
+describe(convertJsxBanProps, () => {
+    test("conversion without arguments", () => {
+        const result = convertJsxBanProps({
+            ruleArguments: [],
+        });
+
+        expect(result).toEqual({
+            rules: [
+                {
+                    ruleName: "react/forbid-component-props",
+                },
+            ],
+            plugins: ["eslint-plugin-react"],
+        });
+    });
+
+    test("conversion with a single prop", () => {
+        const result = convertJsxBanProps({
+            ruleArguments: [["someProp"]],
+        });
+
+        expect(result).toEqual({
+            rules: [
+                {
+                    ruleArguments: [
+                        {
+                            propName: "someProp",
+                        },
+                    ],
```

Comment:
Not 100% confident about the correct object in this case as the docs states that it is an array of props or an object with specific options, but both cases are inside a `forbid` option 🤔 

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-component-props.md#rule-options

Reply:
Oh hey yes good catch!